scss/_focus.scss

```diff
@@ -1,6 +1,6 @@
 @mixin focus-default {
   box-shadow: none;
-  outline: 0.25rem solid $focus-color;
+  outline: 0.25rem solid var(--theme-focus-color);
   outline-offset: 0.25rem;
 }
```

Comment:
Should this be in the mixins file?

Reply:
we chose to put focus states in their own file so we can override and keep them together. some UI things need special attention to make their focus indicators similar to other UI things’ focus indicators